scrud-nuxt/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
Should this file be committed?

Reply:
Yes, production openteams-nuxt uses it and it is super useful while developing. It is supported by most editors and helps with styling the code. [editorconfig.org](https://www.editorconfig.org).

Reply:
@dcharbon